install_config/upgrading/automated_upgrades.adoc

```diff
@@ -278,3 +278,8 @@ common issues:
 [Note] Completed with no errors or warnings seen.
 ----
 ====
+
+Verify in your old *_/etc/sysconfig/openshift-master_* and
+*_/etc/sysconfig/openshift-node_* files that any custom configuration is added
+to your new *_/etc/sysconfig/atomic-openshift-master_* and
+*_/etc/sysconfig/atomic-openshift-node_* files.
```

Comment:
Sorry, I forgot to mention that the context of this is that it's only something you'll need to check when going from either:

OSE 3.0 --> 3.1

or

Origin 1.0 --> 1.1

Those were the upgrade paths that introduced these file name changes, but it was slightly different for OSE vs Origin. For OSE, they went from:

/etc/sysconfig/openshift-master --> /etc/sysconfig/atomic-openshift-master
/etc/sysconfig/openshift-node --> /etc/sysconfig/atomic-openshift-node

For Origin they went from:

/etc/sysconfig/openshift-master --> /etc/sysconfig/origin-master
/etc/sysconfig/openshift-node --> /etc/sysconfig/origin-node

So we'll need to 1) add something like "If you upgrade from (OSE 3.0 or Origin 1.0) to (OSE 3.1 or Origin 1.1), verify these files" and 2) make sure the file references are Origin/OSE appropriate.